plugins/org.jkiss.dbeaver.ui.editors.data/src/org/jkiss/dbeaver/ui/controls/resultset/view/ErrorPresentation.java

```diff
@@ -98,29 +98,6 @@ public void createPresentation(@NotNull IResultSetController controller, @NotNul
                 textWidget.setText(sqlText);
             }
         }
-
-        try {
```

Comment:
Let's make it conditional (e.g. static variable REMEBER_SASH_RATIO). May be useful if we will change our mind after release.

Reply:
Proposed varaible added to keep state.

Reply:
I believe we should keep try catch too in this case

Reply:
Nobody, throw Exception here, specification of weight same always in integer by internal contract api.

Reply:
OK